Makefile.am

```diff
@@ -24,7 +24,7 @@ install-data-hook: install-systemd-service
 uninstall-hook: uninstall-systemd-service
 
 usbguard_notifier.service: $(top_srcdir)/usbguard_notifier.service.in
-		$(SED) -e "s|%sbindir%|${sbindir}|" -e "s|%sysconfdir%|${sysconfdir}|" $^ > $@ || rm -f $@
+		$(SED) -e "s|%bindir%|${sbindir}|" $^ > $@ || rm -f $@
```

Comment:
shouldnt there be ${bindir} instead of ${sbindir} ?

Reply:
yes